spec/models/manual_spec.rb

```diff
@@ -20,15 +20,15 @@
   }
 
   let(:id) { "0123-4567-89ab-cdef" }
-  let(:updated_at) { double(:updated_at) }
-  let(:originally_published_at) { double(:originally_published_at) }
-  let(:use_originally_published_at_for_public_timestamp) { double(:use_originally_published_at_for_public_timestamp) }
-  let(:title) { double(:title) }
-  let(:summary) { double(:summary) }
-  let(:body) { double(:body) }
-  let(:organisation_slug) { double(:organisation_slug) }
-  let(:state) { double(:state) }
-  let(:slug) { double(:slug) }
+  let(:updated_at) { Time.parse("2001-01-01") }
+  let(:originally_published_at) { DateTime.parse("2002-02-02") }
```

Comment:
[`ManualRecord::Edition#updated_at` is defined by `Mongoid::Timestamps`](https://github.com/alphagov/manuals-publisher/blob/56aae0810219bbc119c28698a8b4663fc050672b/app/models/manual_record.rb#L71) whereas [`originally_published_at` is has a custom definition as `DateTime`](https://github.com/alphagov/manuals-publisher/blob/56aae0810219bbc119c28698a8b4663fc050672b/app/models/manual_record.rb#L80). My guess is the developer either didn't realise they were different or had a specific reason to make them different.

I consciously chose not to address this problem in this PR, but I have have just captured it in #1126.